cts/cli/regression.rules.exp

```diff
@@ -17,7 +17,12 @@ Error checking rule: No such device or address
       <primitive class="ocf" id="dummy" provider="heartbeat" type="Dummy"/>
     </resources>
     <constraints>
-      <rsc_location id="no-date-expression" rsc="dummy" score="-INFINITY" node="node01"/>
+      <rsc_location id="cli-too-many-date-expressions" rsc="dummy">
+        <rule id="cli-rule-too-many-date-expressions" score="INFINITY" boolean-op="or">
+          <date_expression id="cli-date-expression-1" operation="gt" start=""/>
+          <date_expression id="cli-date-expression-2" operation="lt" end=""/>
+        </rule>
+      </rsc_location>
```

Comment:
Sorry, I'm not quite following here.

Reply:
A test for a rule without any date_expression is still a good idea.

A &lt;rule&gt; can contain any combination of &lt;expression&gt; &lt;date_expression&gt; and (sub-) &lt;rule&gt; elements. &lt;expression&gt; checks node attributes:

https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html-single/Pacemaker_Explained/index.html#idm47160816617536

I didn't think about sub-rules before, we should use "//" instead of "/" when xpath'ing for date_expression.